critiques/pr-29/check_comments.py

```
@@ -0,0 +1,82 @@
+    Parameters:
+        path (pathlib.Path): the file to tokenize.
+
+    Returns:
+        list: the comment token texts.
+    """
+
+    found = []
+    with open(path, 'rb') as handle:
+        for token in tokenize.tokenize(handle.readline):
+            if token.type == tokenize.COMMENT:
+                found.append(token.string.rstrip())
```

Ruff (0.16.1)

[warning] Use `list.extend` to create a transformed list

(PERF401)